course/pages/dagster-essentials/lesson-3/2-defining-your-first-asset.md

```diff
@@ -10,13 +10,58 @@ In this course, you’ll use data from [NYC OpenData](https://opendata.cityofnew
 
 Your first asset, which you’ll name `taxi_trips_file`, will retrieve the yellow taxi trip data for March 2023 and save it to a location on your local machine.
 
-1. First, navigate to and open the `assets/trips.py` file in your Dagster project. This is where you’ll write your asset code.
+## Project structure
 
-2. At the top of the `trips.py` file, add the following imports:
+Before we write our first asset, let's talk a little about project structures in Dagster. In the previous lesson we mentioned `dg` and how it offers a lot of helpful functionality to quickstart our project. We can use commands like `dg scaffold project` to initialize a `uv` virtual environment for us but we already took care of that when we set up the course in lesson 2.
```

Comment:
The `uv` virtual environment is optional now, I think a user has to `dg scaffold project`, then `uv venv`.

Reply:
I think all the `uv` instructions should get reworked and standardized as when we get closer to release.